holopy/scattering/theory/lens.py

```diff
@@ -0,0 +1,192 @@
+import numpy as np
+
+try:
+    import numexpr as ne
+    NUMEXPR_INSTALLED = True
+except ModuleNotFoundError:
+    NUMEXPR_INSTALLED = False
```

Comment:
It might be nice to get a warning that installing numexpr might speed things up. Right now there's no way to know it's even an option since it silently switches to a fallback.

Reply:
:+1:

Reply:
Okay I'll add a warning in the next revision.